.github/workflows/criteo-cookbooks-ci.yml

```diff
@@ -27,14 +27,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        instance: ['default-centos-7', 'default-ubuntu-1604', 'default-fedora-29']
+        instance: ['python3-centos-7', 'python3-centos-8',
+                   'python3-pypy3-centos-7', 'python3-pypy3-centos-8',
+                   'python3-chef-14-centos-7', 'python3-chef-14-centos-8']
```

Comment:
You keep `python3-chef-14-centos-8` here but you have excluded the `centos-8` platform in the Chef-14 suite due to DNF issue on Chef < 16